src/content/en/updates/2016/10/resizeobserver.md

```diff
@@ -0,0 +1,169 @@
+{# wf_tags: chrome54 #}
+{# wf_featured_image: /web/updates/images/generic/visibility.png #}
+
+# ResizeObserver – It’s like document.onresize for elements {: .page-title }
+
+{% include "web/_shared/contributors/surma.html" %}
+
+After [MutationObserver], [PerformanceObserver] and [IntersectionObserver], we
+got another observer for your collection! `ResizeObserver` allow you to get
+notified when an element’s content rectangle has changed its size and react
+accordingly. The [spec](https://wicg.github.io/ResizeObserver/) is currently
+being iterated on in the WICG and *your* feedback is very much welcome!
+
+## Motivation
+
+So far, you had to attach a listener for the document’s `resize` event to get
+notified of any change of the viewport’s dimensions. In the event handler, you
+would then have to figure out which elements have been affected by that change
+and call the respective routine to react appropriately. If you need the new
+dimensions of an element after a resize, you will need to call
+`getBoundingClientRect` or `getComputerStyle`, which can cause layout thrashing
+if you don’t take care of batching *all* your reads and *all* your writes.
+
+And then you realize that this doesn’t even cover the cases where elements
+change their size without main window having been resized. Appending new
+children, an element’s `display` style being set to `none` or similar actions
+can change the size of an element, its siblings or ancestors.
+
+This is why `ResizeObserver` is a useful primitive. They react to changes in
+size of any of the observed *elements*, independent on what caused that change.
+They provide you access to the new size of the observed elements, too. Let’s get
+straight into it!
+
+## API
+
+All the APIs with the “observer” suffix I mentioned above share a simple API
+design. `ResizeObserver` will fit in just fine. You create a `ResizeObserver`
+object and pass a callback to the constructor. The callback will be given an
+array of `ResizeOberverEntries` – one entry per observed element – which
+containthe new dimensions for the element.
+
+    var ro = new ResizeObserver( entries => {
+      for (let entry of entries) {
+        const cr = entry.contentRect;
+        console.log('Element:', entry.target);
+        console.log(`Element size: ${cr.width}px x ${cr.height}px`);
+        console.log(`Element padding: ${cr.top}px ; ${cr.left}px`);
+      }
+    });
+
+    // Observe one or multiple elements
+    ro.observe(someElement);
+
+## Some details
+
+### What is being reported?
+
+It is important to note that `ResizeObserver` *report* both the dimensions of
+the `contentRect` and the padding, while they only *watch* the `contentRect`.
+The `contentRect` is *not* to be confused with the bounding box of the element.
+Consequently, the top and left attribute of the `contentRect` are the offset
+from the border box. SVGs are an exception to the rule, where the dimensions of
+the bounding box will be reported.
```

Comment:
Can you take another crack at this paragraph? I don't have enough information to know the difference between a contentRect and a bounding box. You never define contentRect and I can't find a formal definition of bounding box that I can link to. (The term is not even in the index of my CSS book.)